requirements.txt

```diff
@@ -0,0 +1,32 @@
+testbook
```

Comment:
I think it is easier for notebook authors in the future if each notebook has its own requirements file.